go.mod

```diff
@@ -97,3 +97,5 @@ require (
 	google.golang.org/protobuf v1.31.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/hashicorp/cap v0.4.1 => /Users/john.landa/Dev/cap
```

Comment:
To be replaced once the Cap PR is merged.

https://github.com/hashicorp/cap/pull/128